cesium/internal/unary/writer.go

```diff
@@ -63,7 +63,7 @@ var (
 		EnableAutoCommit:         config.False(),
 		AutoIndexPersistInterval: 1 * telem.Second,
 	}
-	WriterClosedError = core.EntityClosed("unary.writer")
+	writerClosedError = core.EntityClosed("unary.writer")
```

Comment:
Small convention thing. It seems like you typically prefix your errors with `err` as opposed to a suffix. This is very small, but perhaps something to think about standardizing